src/pages/Settings/Classrooms/LeaveClassroomModal.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Modal from "../../../components/modal_shared/Modal";
+import Header from "../../../components/modal_shared/Header";
+import Heading from "../../../components/modal_shared/Heading";
+import Footer from "../../../components/modal_shared/Footer";
+import ButtonContainer from "../../../components/modal_shared/ButtonContainer";
+import GoToButton from "../../../components/modal_shared/GoToButton";
+
+import * as s from "./LeaveClassroomModal.sc";
+
+export default function LeaveClassroomModal({
+  isLeaveClassroomModalOpen,
+  handleCloseLeaveClassroomModal,
+  currentClassroom,
+  leaveClassroom,
+}) {
+  function handleLeaveClassroom(e) {
+    leaveClassroom(e, currentClassroom);
+    handleCloseLeaveClassroomModal();
+  }
+  return (
+    <Modal
+      open={isLeaveClassroomModalOpen}
+      onClose={handleCloseLeaveClassroomModal}
+    >
+      <Header>
+        <Heading>
+          You are about to leave the following classroom:{" "}
+          <s.ColorAccent>{currentClassroom.name}</s.ColorAccent>
```

Comment:
While I think blue is a nice contrast colour - my first intuition was that I could click it as a link. This might just be me and Blue = Link association, but just a not.

Reply:
Good observation! I'll change it

Reply:
For now, I've decided to go with dark orange (gold) as the brighter one is challenging to read (not enough contrast).
I think in the future we need to consider adding more shades in between these colors (similarly as with blue) for improved variety and contrast in some instances (e.g. buttons). 

| Orange (not selected) | Dark orange (selected) |
|----------|----------|
|  <img width="666" alt="Screenshot 2024-09-11 at 10 41 37" src="https://github.com/user-attachments/assets/538f7bc8-cd6d-42dd-a433-f55408eb9c5a"> |  <img width="666" alt="Screenshot 2024-09-11 at 10 41 14" src="https://github.com/user-attachments/assets/7cf8dbe7-9b18-4304-a902-416e6ccf2df3"> |

Reply:
I agree with the bolder option!